src/bootstrap/builder.rs

```diff
@@ -884,7 +884,11 @@ impl<'a> Builder<'a> {
             rustflags.arg("-Zforce-unstable-if-unmarked");
         }
 
-        rustflags.arg("-Zexternal-macro-backtrace");
+        if stage == 0 {
```

Comment:
We probably want a FIXME comment here so it is caught when the next stage promotion happens.

Reply:
Best way is a `cfg(bootstrap)` containing comment, since that's what I search for.

Reply:
Unlike some other things, it would outright error so this is less important.
But I think I'll do what @Mark-Simulacrum said.